projects/go-tester/src/app/app.module.ts

```diff
@@ -93,7 +94,8 @@ import { AppGuard } from './app.guard';
     GoTableModule,
     GoTextAreaModule,
     GoToastModule,
-    GoToasterModule
+    GoToasterModule,
+    GoHeaderBarModule
```

Comment:
Can this be alphabetized, please!

Reply:
In case you need help: https://www.youtube.com/watch?v=qTvhKZHAP8U

Reply:
I watched it 6 times but the whole ACDBs still confuse me